openssl/src/lib.rs

```diff
@@ -177,18 +177,28 @@ pub mod memcmp;
 pub mod nid;
 #[cfg(not(osslconf = "OPENSSL_NO_OCSP"))]
 pub mod ocsp;
+#[cfg(ossl300)]
+mod ossl_param;
 pub mod pkcs12;
 pub mod pkcs5;
 #[cfg(not(any(boringssl, awslc)))]
 pub mod pkcs7;
 pub mod pkey;
 pub mod pkey_ctx;
+#[cfg(ossl350)]
+pub mod pkey_ml_dsa;
+#[cfg(ossl350)]
+pub mod pkey_ml_kem;
+#[cfg(ossl350)]
+pub mod pkey_slh_dsa;
 #[cfg(ossl300)]
 pub mod provider;
 pub mod rand;
 pub mod rsa;
 pub mod sha;
 pub mod sign;
+#[cfg(ossl300)]
+pub mod signature;
```

Comment:
Should we use `#[cfg(ossl350)]`  here to avoid the need to add the `#[allow(unused_imports)]` inside? Even though the functions might be available in older versions, they are for no good use as far as I know.